src/main/java/com/denizenscript/ddiscordbot/objects/DiscordChannelTag.java

```diff
@@ -58,11 +58,11 @@ public static DiscordChannelTag valueOf(String string, TagContext context) {
             }
             return null;
         }
-        long chanID = Long.parseLong(string);
-        if (chanID == 0) {

```

Comment:
aaaaaany particular reason you're randomly changing unrelated variable names in files you're otherwise not even editing?

Reply:
That was originally just an overview of the files before I started making any changes, not relevant at all to this pr (other than updating deprecated methods). I can revert this commit if needed.